java/code/src/com/redhat/rhn/manager/contentmgmt/FilterTemplateManager.java

```diff
@@ -64,20 +67,22 @@ private FilterTemplateManager(ModulemdApi modulemdApiIn) {
     public List<ContentFilter> createLivePatchFilters(String prefix, PackageEvr kernelEvr, User user) {
         ensureOrgAdmin(user);
 
-        Map<String, FilterCriteria> criteria =
-                Map.of("livepatches", new FilterCriteria(FilterCriteria.Matcher.CONTAINS_PKG_GT_EVR,
-                "package_nevr", "kernel-default " + kernelEvr.toString()),
-                "noreboot", new FilterCriteria(FilterCriteria.Matcher.CONTAINS,
-                "keyword", "reboot_suggested"));
+        HashSet<Triple<String, FilterCriteria, EntityType>> criteria = new HashSet<>();
+        criteria.add(Triple.of("livepatches", new FilterCriteria(FilterCriteria.Matcher.CONTAINS_PKG_GT_EVR,
+                               "package_nevr", "kernel-default " + kernelEvr.toString()), EntityType.ERRATUM));
+        criteria.add(Triple.of("noreboot", new FilterCriteria(FilterCriteria.Matcher.CONTAINS,
+                                "keyword", "reboot_suggested"), EntityType.ERRATUM));
+        criteria.add(Triple.of("noreboot2", new FilterCriteria(FilterCriteria.Matcher.CONTAINS_PROVIDES_NAME,
+                                "package_provides_name", "installhint(reboot-needed)"), EntityType.ERRATUM));
```

Comment:
All the 3rd items are `ERRATUM`, so maybe you can simplify it back to `Map`?

Reply:
In the beginning I had a PACKAGE filter so I changed the Map to a List of Triples.
But I think I will roleback this change again